lua/telescope-nvim.lua

```diff
@@ -67,4 +67,10 @@ M.config = function()
     require("telescope").load_extension("media_files")
 end
 
+function M.search_dootfiles()
+    require('telescope.builtin').find_files {
+        cwd = "~/.config/nvim/",
```

Comment:
@ashincoder @siduck76 ash pointed out that you may need to change the config location.